distortion_coeffs: [0.04740989502423929, -0.0453917568214522, -0.0009495923262117699, 0.0033153366069044767]
intrinsics: [971.2548275969137, 978.730843946176, 756.3142054039969, 552.3661835613533]
resolution: [1440, 1080]